configure.ac

```diff
@@ -85,9 +85,9 @@ AC_REQUIRE_AUX_FILE([tap-driver.sh])
 AC_PATH_PROG([NATURALDOCS], [naturaldocs], [notfound])
 AC_ARG_VAR([NATURALDOCS], [Path to naturaldocs])
 AC_SUBST([GLIB_COMPILE_RESOURCES], [`$PKG_CONFIG --variable glib_compile_resources gio-2.0`])
-AC_PATH_PROG([SCSS], [scss], [notfound])
-AS_IF([test x$SCSS = xnotfound], [AC_MSG_ERROR([SCSS compiler not found.
-Try installing the "sass" Ruby gem or its equivalent package.])])
+AC_PATH_PROG([SCSS], [sassc], [notfound])
```

Comment:
For development tools, it's likely that we'll continue using apt packages at least for a while, until it becomes more convenient to develop within the sandbox. How about making this into `[sassc scss]` so that both compilers are supported for the time being? If not, then we should make sure to make an apt package for `sassc`.